src/Tribe/Utils/Color.php

```diff
@@ -513,4 +513,61 @@ private static function _checkHex( $hex ) {
 
 		return $color;
 	}
+
+	/**
+	 * Get color that contrasts the most.
+	 * 
+	 * @since TBD
+	 *
+	 * @param string $hex_color  Hex value of color to compare.
+	 * @param string $lite_color Hex value of dark contrast color.
+	 * @param string $dark_color Hex value of light contrast color.
+	 * 
+	 * @return string
+	 */
+	public static function get_contrast_color( $hex_color, $lite_color = '#ffffff', $dark_color = '#000000' ) {
```

Comment:
Maybe we could use `$light_color`? Lite would be the informal way to call it, right?

Reply:
I think their interchangeable, but you're right. Light is a little more formal than lite. I used 'lite' for alignment purposes since it was the same number of characters as 'dark'. I'll make the change.